Add global 'use strict' requirements

kjdelisle opened the pull request:
connected to https://github.com/strongloop-internal/scrum-nodeops/issues/1596

Require 'use strict' at the top of all modules.
(Used to be a default rule, but disabled as a default in 2.0)

kjdelisle commented:
@sam-github @rmg PTAL

sam-github commented:
Looks fine to me... but you need to wait for all the dependent repos to build and make sure they pass, we can't update the eslint rules and cause half our ci to go red.

@rmg, ok by you?

sam-github commented:
yes, `'use strict;'` is missing from many files. we should add, but not a red-hot priority for this sprint, lets do it next one

kjdelisle commented:
@slnode test please
re-running to see what repositories would be non-compliant at this time

Commit:
Add global 'use strict' requirements

Require 'use strict' at the top of all modules.
(Used to be a default rule, but disabled as a default in 2.0)

kjdelisle commented:
@slnode test please
Now we're going to see if we've solved the downstream breakage.

kjdelisle commented:
@slnode test please

kjdelisle commented:
@slnode test please

sam-github commented:
@slnode test please

sam-github commented:
LGTM
